pkg/autoscaler/autoscaler/autoscaler_manager.go

```diff
@@ -125,9 +128,44 @@ func (am *autoScalerManager) syncTidbClusterReplicas(tac *v1alpha1.TidbClusterAu
 	return nil
 }
 
-//TODO: sync tac status
-func (am *autoScalerManager) syncAutoScalingStatus(tc *v1alpha1.TidbCluster, oldTc *v1alpha1.TidbCluster,
+func (am *autoScalerManager) updateAutoScaling(oldTc *v1alpha1.TidbCluster,
 	tac *v1alpha1.TidbClusterAutoScaler) error {
+	if tac.Annotations == nil {
+		tac.Annotations = map[string]string{}
+	}
+	f := func(key string) (string, error) {
+		v, ok := tac.Annotations[key]
+		if ok {
+			ts, err := strconv.ParseInt(v, 10, 64)
+			if err != nil {
+				klog.Errorf("failed to convert label[%s] key to int64, err:%v", key, err)
+				return "", err
+			}
+			return time.Unix(ts, 0).Format(time.RFC3339), nil
+		}
+		return "", nil
+	}
+
+	if tac.Spec.TiKV != nil {
+		tac.Status.TiKV.CurrentReplicas = oldTc.Status.TiKV.StatefulSet.CurrentReplicas
+		lastTimestamp, err := f(label.AnnTiKVLastAutoScalingTimestamp)
+		if err != nil {
+			return err
+		}
+		tac.Status.TiKV.LastAutoScalingTimestamp = &lastTimestamp
```

Comment:
If `lastTimeStamp==""`, do we still need to set it here?

Reply:
updated.